cldfbench_nominalperson_cldf.py

```diff
@@ -22,26 +23,31 @@ def cmd_download(self, args):
         pass
 
     def cmd_makecldf(self, args):
-        """
-        Convert the raw data to a CLDF dataset.
-
-        >>> args.writer.objects['LanguageTable'].append(...)
-        """
-        
-        with Writer(cldf_spec) as writer:
-            for row in self.raw_dir.read_csv(
-                'examples.csv',
-                dicts=True, 
-            ):
-                writer.objects['ExampleTable'].append({
+        # Extend the schema:
+        args.writer.cldf.add_component('LanguageTable')
+        args.writer.cldf.add_component('ExampleTable')
+
+        for lid, rows in itertools.groupby(
+            sorted(
+                self.raw_dir.read_csv('examples.csv', dicts=True),
+                key=lambda row: (row['Language_ID'], int(row['ID']))),
+            lambda row: row['Language_ID']
+        ):
+            rows = list(rows)
+            args.writer.objects['LanguageTable'].append(dict(
+                ID=lid,
+                Name=rows[0]['Language_Name'],
+            ))
+            for row in rows:
+                args.writer.objects['ExampleTable'].append({
                     'ID': row['ID'],
                     'Language_ID': row['Language_ID'],
                     'Primary_Text': row['Primary_Text'],
-                    'Analyzed_Word': row['Analyzed_Word'],
-                    'Gloss': row['Gloss'],
+                    'Analyzed_Word': row['Analyzed_Word'].split(),
+                    'Gloss': row['Gloss'].split(),
```

Comment:
Analyzed_Word and Gloss are expected to be list-valued. (And the two columns are expected to have the same number of items.)